packages/ember-debug/lib/handlers.js

```diff
@@ -0,0 +1,27 @@
+import isPlainFunction from 'ember-debug/is-plain-function';
+
+export let HANDLERS = { };
```

Comment:
I believe you need to evict these somehow in tests, otherwise you will retain deprecations across app and isolated container instances.